homeassistant/components/coronavirus/sensor.py

```diff
@@ -5,14 +5,21 @@
 from . import get_coordinator
 from .const import ATTRIBUTION, OPTION_WORLDWIDE
 
+SENSORS = {
+    "confirmed": "mdi:emoticon-neutral-outline",
+    "current": "mdi:emoticon-frown-outline",
+    "recovered": "mdi:emoticon-happy-outline",
+    "deaths": "mdi:emoticon-dead-outline",
+}
+
 
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Defer sensor setup to the shared sensor module."""
     coordinator = await get_coordinator(hass)
 
     async_add_entities(
         CoronavirusSensor(coordinator, config_entry.data["country"], info_type)
-        for info_type in ("confirmed", "recovered", "deaths", "current")
+        for info_type in SENSORS.keys()
```

Comment:
Keys is the default iterator.
```suggestion
        for info_type in SENSORS
```